packages/ndla-accordion/src/accordionUtil.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { ReactElement, ReactNode, Children } from 'react';
+import AccordionSection, { Props as ASProps } from './AccordionSection';
```

Comment:
jeg foretrekker at vi skriver ut ord og ikke forkorter de
```suggestion
import AccordionSection, { Props as AccordionSectionProps } from './AccordionSection';
```